source/install/install-rhel-8.rst

```diff
@@ -20,18 +20,146 @@ You can also use these instructions to install Mattermost on CentOS 8 or Oracle
+   
+            The first time that you start MySQL:
+        
+            - The superuser account ``'root'@'localhost'`` is created with a password. Get this password by running ``sudo grep 'temporary password' /var/log/mysqld.log``. 
+            - The ``validate_password`` plugin is installed. The plugin forces passwords to contain at least one upper case letter, one lower case letter, one digit, and one special character, and that the total password length is at least eight characters.
+
+    6. Change the root password. 
+
+    7. Set MySQL to start automatically when the machine starts by running ``sudo systemctl enable mysqld``.
+
+    8. Create the Mattermost user *mmuser* by running ``mysql> create user 'mmuser'@'%' identified by 'mmuser-password';``.
+
+        .. note::
+        
+            - Use a password that's more secure than ``mmuser-password``.
+            - The ``%`` means that mmuser can connect from any machine on the network. However, it's more secure to use the IP address of the machine that hosts Mattermost. For example, if you install Mattermost on the machine with IP address ``10.10.10.2``, then use the following command: ``mysql> create user 'mmuser'@'10.10.10.2' identified by 'mmuser-password';``.
+
+    9. Create the Mattermost database by running ``mysql> create database mattermost;``.
+
+    10. Grant access privileges to the user mmuser by running ``mysql> grant all privileges on mattermost.* to 'mmuser'@'%';``.
+  
+        .. note::
+    
+            This query grants the MySQL user all privileges on the database for convenience. If you need more security, you can use the following query to grant the user only the privileges necessary to run Mattermost: ``mysql> GRANT ALTER, CREATE, DELETE, DROP, INDEX, INSERT, SELECT, UPDATE, REFERENCES ON mattermost.* TO 'mmuser'@'%';``.
+    
+    11. Log out of MySQL by running ``mysql> exit``.
+
+    With the database installed and the initial setup complete, you can now install the Mattermost server.
+
+  .. tab:: Install PostgreSQL
+
+    Install and set up a PostgreSQL database for use by the Mattermost server. 
+
+    1. Log in to the server that will host the database, and open a terminal window.
+
+    2. Install PostgreSQL. See the `PostgreSQL <https://www.postgresql.org/download/linux/redhat/>`__ documentation for details. 
+
+    3. Initialize the database by running ``sudo postgresql-setup initdb``.
+
+    4. Set PostgreSQL to start on boot by running ``sudo systemctl enable postgresql``.
+
+    5. Start the PostgreSQL server by running ``sudo systemctl start postgresql``.
+
+    6. Switch to the *postgres* Linux user account that was created during the installation by running ``sudo -iu postgres``.
+
+    7. Start the PostgreSQL interactive terminal by running ``psql``.
+
+    8.  Create the Mattermost database by running ``postgres=# CREATE DATABASE mattermost WITH ENCODING 'UTF8' LC_COLLATE='en_US.UTF-8' LC_CTYPE='en_US.UTF-8' TEMPLATE=template0;``.
```

Comment:
Any reason this is different from the Ubuntu command? Ubuntu is just `CREATE DATABASE mattermost`

Reply:
@nab-77 - Any idea why we have such an elaborate command documented?

Reply:
I would need to look into the defaults variance between Ubuntu and RHEL, but right now I don't have the bandwith. Immediate thoughts are:

1. Its been in the docs for some time and so its not causing an issue right now.
2. We don't really need to provide this level of detail here. Simple the DB and any specific tuning/config for said DB.

@cwarnermm the ongoing review will remove any redundant information. For 7.2 I see no impact on leaving this as is.